localization/fi.ts

```diff
@@ -425,7 +425,7 @@
     <message>
         <location filename="../qml/pages/SettingsPage.qml" line="132"/>
         <source>Route search parameters</source>
-        <translation>Reitinhaku parametrit</translation>
+        <translation>Reittihaku</translation>
```

Comment:
You need to run lupdate to update the translation files when changing English strings in source files.